test/programs/TestComments.pebl

```diff
@@ -0,0 +1,5 @@
+#comment at beginning of file (no space)
```

Comment:
I like your test case of comment as the last line. Let's add a case where the indent of a comment is changes as well

Reply:
added